init the features indexer

Omarabdul3ziz opened the pull request:
- modify the node response to have the supported features
- add the features indexer to the list of indexers health
- support filtering nodes based on its features
- support filtering farms based on its nodes' features
- add the indexer work that calls the node and decide its have a light network version or the normal one
- decide a full day as the interval for the features indexer
- update swagger docs with the new filters
- add random value generators for the new filters
- add the test conditions for the new filter tests
- add the new table to schema and add a dump generator and loader
- https://github.com/threefoldtech/tfgrid-sdk-go/issues/1209

### Checklist

- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstring

Omarabdul3ziz committed:
init the features indexer:
- modify the node response to have the supported features
- add the features indexer to the list of indexers health
- support filtering nodes based on its features
- support filtering farms based on its nodes' features
- add the indexer work that calls the node and decide its have a light network version or the normal one
- decide a full day as the interval for the features indexer
- update swagger docs with the new filters
- add random value generators for the new filters
- add the test conditions for the new filter tests
- add the new table to schema and add a dump generator and loader

xmonader commented:
![image](https://github.com/user-attachments/assets/0967f60a-2d08-4104-bfc0-e3271e5d49ec)

Can we discuss? Seems that we only care here about wg, ygg, public ip only? afaik there're changes for other workloads

I was expecting that we will have the version and the node features (all of them) indexed somehow

Omarabdul3ziz committed:
refactor the feature indexer to include all features:
- add validation on the nodefeature filter values
- add the features filter/field to the node and farm getters methods
- update the feature indexer parser to include the feature set to the report
- update swagger docs
- fix the nullable features to be an empty slice instead
- add the feature indexer to health state
- add a helper methods file
- update schema/generator/loader/tests

Omarabdul3ziz committed:
chores:
- add enhance notes
- create function for common checks
